ui/components/app/confirm-page-container/confirm-page-container-header/confirm-page-container-header.component.js

```diff
@@ -64,7 +75,23 @@ export default function ConfirmPageContainerHeader({
             </span>
           </div>
         )}
-        {isFullScreen ? null : <NetworkDisplay />}
+        {isFullScreen ? null : (
+          <PickerNetwork
+            label={
+              networkType === NETWORK_TYPES.RPC
+                ? nickname ?? t('privateNetwork')
+                : t(getNetworkLabelKey(networkType))
+            }
+            backgroundColor={BackgroundColor.transparent}
+            borderColor={BorderColor.borderMuted}
+            iconProps={{ display: 'none' }}
+            avatarNetworkProps={{
+              backgroundColor: testNetworkBackgroundColor,
+              className: 'confirm-page-container-header-avatar-network-base',
+            }}
+            className="mm-picker-network-div"
```

Comment:
We can remove this className as we shouldn't need it now that the element has been changed to a div
```suggestion
```

Reply:
Removed